### DISCUSSION TITLE
Stables bonus

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#3520] [Q&A]
What is the bonus for movement from Stables? In the code I see that we add 400 and 500 which is wrong.

**Comment by LeHerosInconnu** [comment]
Hello @ihhub,

> What is the bonus for movement from Stables? In the code I see that we add 400 and 500 which is wrong.

In the original game, the hero receives a bonus of 400 movement points as soon as he visits the Stables, and he keeps this bonus of 400 movement points until the end of the week.

Related discussion: https://github.com/ihhub/fheroes2/discussions/3139.
